scripts/generator/templates/template-release-notes.md

```diff
@@ -12,19 +12,25 @@ Vaadin 14 is an LTS (long term support) version, which will be supported at leas
 Vaadin also provides [commercial support and warranty](https://vaadin.com/support).
 
 ## New and Noteworthy
-- New theme variants for Notification
+
+Here are the highlighted new and improved features in Vaadin 14. To see the full list of bug fixes and improvements, check Included Projects and Change Log.
+
+### Framework
+- The possibility to use Polymer 3, npm and JavaScript Modules instead of Polymer 2, Bower and HTML Imports.
+- Adding @PreserveOnRefresh on a router layout or route will preserve the user edited component state when the user refreshes the page by reusing the component instances
+- Embedding Flow applications is now possible by exporting it as a web component.
```

Comment:
nit: "applications" (plural), but "exporting it as a web component" (singular)

Reply:
Changed applications => application